api/controllers/accounts.js

```diff
@@ -21,16 +21,23 @@ var swaggerHelper = require('../../helpers/swagger');
 var modules;
 
 /**
- * Initializes with scope content and private variables:
- * - modules
- * @class AccountsController
- * @classdesc Main System methods.
- * @param {scope} scope - App instance.
+ * Initializes with scope content and private variable: modules.
+ * @module controllers/accounts
+ * @requires lodash
+ * @requires module:helpers/apiError
+ * @requires module:helpers/swagger.generateParamsErrorObject
+ * @param {Object} scope - App instance.
  */
 function AccountsController (scope) {
 	modules = scope.modules;
 }
 
+/**
+ * @public
```

Comment:
Do we want to specify this for every function or just opt out with `@private` and assume public by default?

Reply:
jsdoc default public: JSDoc treats all symbols as public

Reply:
agreed, `@public` is not necessary as it is the default